app/javascript/packages/form-steps/form-steps.tsx

```diff
@@ -237,8 +237,19 @@ function FormSteps({
       const { element, isRequired } = fields.current[key];
       const isActive = !!element;
 
-      if (isActive && isRequired && !values[key]) {
-        result = result.concat({ field: key, error: new RequiredValueMissingError() });
+      let error: Error | undefined;
```

Comment:
this is the same right?
```suggestion
      let error: Error?;
```

Reply:
No, unfortunately not. I believe the question mark "possibly undefined" is only valid for interfaces or parameters. In this context, it's perceived as the JSDoc nullable type, and not allowed.

![image](https://user-images.githubusercontent.com/1779930/164228618-f307c7c0-2cb5-4ea8-acca-29ba6a3db119.png)

Reply:
bummer!